index.rst

```diff
@@ -1,58 +1,152 @@
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+   The process is complete when the relevant AD(s) and/or Director reviews the note and the note is published (merged to master).
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+   **Engineering review:** Michael Reuter
 
-     Caption text.
+   **Management review:** Wil O'Mullane, Chuck Claver and Bob Blum
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+.. Add content here.
 
-   Feel free to delete this instructional comment.
+Scope
+=====
 
-:tocdepth: 1
+The scope of this Delivery Note is usage of the Rubin Science Platform (including) associated infrastructure for observatory operations, including (but not limited to) control of the observing queue, analysis and visualisation of incoming telescope data and analsysis of telemetry and other quantities stored in the Engineering Facilities Database.
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+The usage of the RSP in this way is "off-requirements".
+The Construction project had a need for interim tooling at the summit pending delivery of other planned components, and so the RSP (in particularly the notebook service nublado) was opportunistically offered as a way to fill in that gap.
+After more than a year of operating in this mode, it has become apparent that the summit RSP is here to stay as it provides both useful tools for the ad-hoc analysis of observatory data as well as deployment infrastructure that can be utilised by other observatory components.
 
-.. sectnum::
+Specifically, the scope is the operation and support of Science Platform clusters in what is informally referred to as the "telescope" (as opposed to "science") deployments:
 
-.. TODO: Delete the note below before merging new content to the master branch.
+* At the summit
+* At the La Serena Base
+* At the various engineering Teststands (currently at NCSA and Tucson)
 
-.. note::
 
-   **This technote is not yet published.**
+Artifacts (code, docs, infrastructure) involved in this delivery
+----------------------------------------------------------------
+
+The usage of the RSP for the telescope deployments is largely "as-is" in terms of features and services.
+No new requirements or usecases have been placed on the feature list as a result, although we have had to make some infrastructure changes to allow it to be effective in the summit/engineering environments.
+
+None of the data publication services (API service) are in use at the telescope deployments.
+The services exercised heavily are:
+
+- The notebook aspect (https://github.com/lsst-sqre/nublado2)
+- The portal aspect and the related Firefly JupyterLab plugin
```

Comment:
Might be worth parenthetically noting that this is the Portal without TAP support, although I suppose that's covered above by no data publication services. I say this just because GPDF wants us to deploy a different container at the T&S deployments that doesn't include the TAP code

```suggestion
- The portal aspect (without TAP integration) and the related Firefly JupyterLab plugin
```

Reply:
Ah I see what you're saying, I will rephrase somewhat so that people don't think there's no TAP from the portal to public TAP servers.